classes/ThemeScanner.php

```diff
@@ -24,7 +25,20 @@ public static function scan()
     {
         $obj = new static;
 
-        return $obj->scanForMessages();
+        $obj->scanForMessages();
+
+        /**
+         * @event translate.themeScanner.afterScan
+         * Fires after theme scanning.
+         *
+         * Example usage:
+         *
+         *     Event::listen('translate.themeScanner.afterScan', function (ThemeScanner $scanner) {
+         *         // added an extra scan. Add generation files...
+         *     });
+         *
+         */
+        Event::fire('translate.themeScanner.afterScan', [$obj]);
```

Comment:
Event should be `rainlab.translate.themeScanner.afterScan`. Also you need to continue to return the result of the scan method

Reply:
@LukeTowers I was going to mention the return thing too, but the method returns `void`, so there was no need for a `return` call in the first place.